### ISSUE TITLE
Where is the Select event for the ToolStripMenuItem?

### ISSUE THREAD

**Issue by rLindorfer** [dotnet/winforms#3572]
# The MenuItem.Select event of the MenuItem is missing in the ToolStripMenuItem

This event should be typically raised when the user places the mouse pointer over the menu item. The event can also be raised when the user highlights a menu item using the keyboard by scrolling to the menu item with the arrow keys. 
https://docs.microsoft.com/en-us/dotnet/api/system.windows.forms.menuitem.select?view=netframework-4.8

# General

When .NET Core 3.1 was introduced, there was a breaking change in using ContextMenus. MenuItem should be replaced by the ToolStripMenuItem. https://docs.microsoft.com/en-us/dotnet/core/compatibility/winforms

However, when I try to replace MenuItem with the ToolStripMenuItem, it is not possible for me to get feedback when the user selects an item using the keyboard.

Some workarounds are the following events: MouseEnter, MouseLeave, MouseHover, etc.
But this does not cover the selection of a ToolStripMenuItem via keyboard (e.g. arrow keys)!

https://social.msdn.microsoft.com/Forums/en-US/39d5376c-a4c7-4136-b0d1-9193dc215ff8/where-is-select-event-of-toolstripmenuitem
https://stackoverflow.com/questions/464493/net1-menuitem-vs-net2-toolstripmenuitem-select-event

**Comment by rLindorfer** [comment]
Could you please give me a brief update concerning my issue?

**Comment by RussKie** [comment]
It is on the backlog.

**Comment by dreddy-work** [comment]
@vladimir-krestov  , Can you take a look at this and see if there is a way to activate tooltips here?

**Comment by dreddy-work** [comment]
@rLindorfer, This has been around and hasn't got traction from the community. If this is still blocking you, Would you be interested in a API proposal as well as the implementation here? We will help you drive the rest of the process. Otherwise, We will not get to it in .NET 7.0

**Comment by Tanya-Solyanik** [comment]
Suggested name `Selected` per [Names of Type Members - Framework Design Guidelines | Microsoft Learn](https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members)

**Comment by v-elnovikova** [comment]
Suggested API proposal: [#81477](https://github.com/dotnet/runtime/issues/81477)

**Comment by rLindorfer** [comment]
@v-elnovikova 
Thank you for your proposal, LGTM and would be useful for so many users!

**Comment by rajeshnandwani1969** [comment]
Please add this event ASAP, I am migrating my project from .nf4 to .net6
